### PR TITLE
Resolve typescript files using webpack config

### PR DESCRIPTION
**Pull request by MaskyS**
build will fail if our code contains any .ts imports. This is because [Webpack does not look for .ts files by default](https://stackoverflow.com/questions/43595555/webpack-cant-resolve-typescript-modules)

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -125,8 +125,14 @@ TypeScript and Babel configs conflict each other, so you can only use one of the
 		exclude: /node_modules/
 	},
 	```
+3. Ask webpack to [resolve ts modules/imports](https://stackoverflow.com/questions/43595555/webpack-cant-resolve-typescript-modules) by adding this to your `webpack.config.json`:
+	```js
+	resolve: {
+	    extensions: ['.ts', '.js', '.json']
```
Comment (notlmn) [inline readme.md]: Let's ignore `.json` here as that is not used/related in this case.

Also use tabs instead of spaces for consistent indentation.

File: readme.md [hunk 2]
```diff
@@ -125,8 +125,14 @@ TypeScript and Babel configs conflict each other, so you can only use one of the
...
+	}
+	```
 
-1. Use the following as `tsconfig.json`, uses [sindresorhus/tsconfig][link-tsconfig] (install it as dependecy before using).
+4. Use the following as `tsconfig.json`, uses [sindresorhus/tsconfig][link-tsconfig] (install it as dependecy before using).
```
Comment (notlmn) [inline readme.md]: Unrelated change, I do prefer having all numbers spelled out (in plain text files), but using version control and changes like this adding entries in between requires all other list items to be modified as well.

Change this and the other entries above back to `1.`

**Review by notlmn — CHANGES_REQUESTED** [review]
Expect for some nit-picks, everything else looks cool good to me

File: readme.md (hunk 2 above)
Comment (MaskyS) [reply readme.md]: ✅

File: readme.md (hunk 1 above)
Comment (MaskyS) [reply readme.md]: ✅

**Comment by notlmn** [comment]
Thanks for the PR @MaskyS 🙌
